pkg/utils/utils.go

```diff
@@ -118,6 +130,16 @@ func SyncNodeState(newState *sriovnetworkv1.SriovNetworkNodeState) error {
 					glog.Errorf("SyncNodeState(): fail to config sriov interface %s: %v", iface.PciAddress, err)
 					return err
 				}
+				if ifaceStatus.Name != "" {
+					if devLink, err := netlink.LinkByName(ifaceStatus.Name); err == nil {
```

Comment:
same as above.

also a helper method of : getLinkType() would reduce code duplication